_episodes/00-setting-the-scene.md

```diff
@@ -57,23 +57,18 @@ are transferable to other similar tools and programming languages.
 
 The course is organised into the following sections:
 
-{% comment %}
-![Course overview diagram](../fig/course-overview.png){: .image-with-shadow width="800px" }
-{% endcomment %}
 
 ![Course overview diagram](../fig/course-overview.svg){: .image-with-shadow width="1000px" }
 {% comment %}
 flowchart LR
-A(1. Setting up
-software environment)
---> B(2. Verifying
-software correctness)
---> C(3. Software development
-as a process)
---> D(4. Collaborative
-development for reuse)
---> E(5. Managing software
-over its lifetime)
+A[1. Setting up
+software environment]
+--> B[2. Verifying
+software correctness]
+--> C[3. Software development
+as a process]
+--> D[4. Collaborative
+development for reuse]
```

Comment:
Out of curiosity: why the change from `()` to `[]`?

Reply:
The way it was it didn't work. And this works but it creates the ticks. I am content with not fully understanding at this moment 🙈😬

Reply:
Fair enough 🙂